### PR TITLE
Issue 103: Create documentation for contributing to documentation

### PR DESCRIPTION
**Pull request by saurabh-vijayvargiya**
@joshmiller83 Created documentation for contributing to documentation

### CODE REVIEW

**Comment by saurabh-vijayvargiya** [comment]
@joshmiller83 Pull request updated. Please review.

File: v2/getting-started/contributing/contributing-to-documentation.rst [hunk 1]
```diff
@@ -11,9 +11,83 @@ issue, go through `the open issues`_, pick one you like and **assign it to you**
...
+| Test 2         | pass          |
++----------------+---------------+
+| Test 3         | fail          |
++----------------+---------------+
+| Test 4         | pass          |
++----------------+---------------+
+
+**For adding tabs**
+
+.. tabs::
```
Comment (joshmiller83) [inline v2/getting-started/contributing/contributing-to-documentation.rst]: We need the same thing you did for the table example for the tab example.

File: v2/getting-started/contributing/contributing-to-documentation.rst [hunk 2]
```diff
@@ -31,8 +105,22 @@ Once you’re done with documenting, push your commits to your fork:
...
+To get the details about platform-based build for your pull request. Click on the ``details`` link.
+
+.. figure:: images/pull-request-2.png
+
+**Note:** Only first 10 open PRs gets the full build.
+
+**Getting feedback**
+
+To get the feedback on your pull request you need to ask somebody to review it. And `Drupal Slack`_ member are ready to review the pull requests for you. But first you will need an invite, you can get invite by submitting your email id here.
+
```
Comment (joshmiller83) [inline v2/getting-started/contributing/contributing-to-documentation.rst]: Remove extra empty line.

**Comment by mglaman** [comment]
Also, looks like it needs to be added to the index http://pr-118-p7wj3jy-boljpk23hsit6.eu.platform.sh/v2/getting-started/contributing/index.html#

**Comment by mglaman** [comment]
Also, general question: Should this be under the v2 section, or the general introduction?

**Comment by saurabh-vijayvargiya** [comment]
@mglaman I think this should go in general introduction, as this documentation applies for both v1 and v2. However this is my personal opinion. What do you think @joshmiller83

**Comment by mglaman** [comment]
I had envisioned under Introduction since it isn't directly related to v2 (Mostly. But it's meta of the documentation itself.)

**Comment by joshmiller83** [comment]
Yes, @saurabh-vijayvargiya please move this page to under this page: http://docs.drupalcommerce.org/introduction.html

**Comment by saurabh-vijayvargiya** [comment]
@joshmiller83 @mglaman Moved to introduction section as per feedback.

**Comment by mglaman** [comment]
Amazing, thanks!

**Pull request merged**